tenants/ai-example/single-model-serving/base/flan-t5-small/test-flan-model/2_grpc_request.ipynb

```diff
@@ -0,0 +1,116 @@
+{
+ "cells": [
```

Comment:
I like the idea of having this in here, but I think it would be better outside of the base folder.  It makes me feel like this is another good reason to break the flan/granite examples out into two separate examples.

What if we did something like this:

```
├── tgis-flan-single-model-server
│   ├── base
│   ├── overlays
│   │   ├── rhoai-eus-2.8
│   │   ├── rhoai-eus-2.8-aws-gpu
│   │   ├── rhoai-fast
│   │   └── rhoai-fast-aws-gpu
│   └── test-notebooks
├── vllm-granite-single-model-server
│   ├── base
│   └── overlays
│       ├── rhoai-eus-2.8-aws-gpu
│       └── rhoai-fast-aws-gpu
```

Reply:
fixed rearranged and refactored